### ISSUE TITLE
[JSX] tags are not aligned when passing jsx to a function

### ISSUE THREAD

**Issue by ipanasenko** [prettier/prettier#2915]
Hi, guys. 

Opening and closing tags are not aligned, but I think they should.
Can you have a look if it's a bug or expected, please?

**Prettier 1.7.1**
[Playground link](https://prettier.io/playground/#%7B%22content%22%3A%22describe('%3CSimpleLink%2F%3E'%2C%20function()%20%7B%5Cn%20%20it('should%20render%20%3Ca%2F%3E%20when%20provided%20absolute%20url'%2C%20function()%20%7B%5Cn%20%20%20%20const%20wrapper%20%3D%20shallow(%5Cn%20%20%20%20%20%20%3CSimpleLink%20href%3D%5C%22http%3A%2F%2Fexample.com%5C%22%3E%5Cn%20%20%20%20%20%20%20%20a%5Cn%20%20%20%20%20%20%3C%2FSimpleLink%3E%5Cn%20%20%20%20)%3B%5Cn%20%20%20%20expect(wrapper).toMatchSnapshot()%3B%5Cn%20%20%7D)%3B%5Cn%7D)%3B%5Cn%22%2C%22options%22%3A%7B%22ast%22%3Afalse%2C%22bracketSpacing%22%3Atrue%2C%22doc%22%3Afalse%2C%22jsxBracketSameLine%22%3Afalse%2C%22output2%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22printWidth%22%3A80%2C%22semi%22%3Atrue%2C%22singleQuote%22%3Afalse%2C%22tabWidth%22%3A2%2C%22trailingComma%22%3A%22none%22%2C%22useTabs%22%3Afalse%7D%7D)

**Input:**
```jsx
describe('<SimpleLink/>', function() {
  it('should render <a/> when provided absolute url', function() {
    const wrapper = shallow(
      <SimpleLink href="http://example.com">
        a
      </SimpleLink>
    );
    expect(wrapper).toMatchSnapshot();
  });
});

```

**Output:**
```jsx
describe("<SimpleLink/>", function() {
  it("should render <a/> when provided absolute url", function() {
    const wrapper = shallow(<SimpleLink href="http://example.com">
        a
      </SimpleLink>);
    expect(wrapper).toMatchSnapshot();
  });
});

```

**Expected behavior:**
Unchanged
```jsx
describe('<SimpleLink/>', function() {
  it('should render <a/> when provided absolute url', function() {
    const wrapper = shallow(
      <SimpleLink href="http://example.com">
        a
      </SimpleLink>
    );
    expect(wrapper).toMatchSnapshot();
  });
});

```

**Comment by ipanasenko** [comment]
BTW, this is introduced in 1.7.1, 1.7.0 worked just fine

**Comment by duailibe** [comment]
This is the same as #2909 and #2910

**Comment by ipanasenko** [comment]
Yeah, looks like #2910 . Thanks

**Comment by suchipi** [comment]
Just released 1.7.2 where this is fixed. Sorry about that!
